src/ucp/core/ucp_request.inl

```diff
@@ -178,7 +178,7 @@ ucp_request_complete_send(ucp_request_t *req, ucs_status_t status)
 static UCS_F_ALWAYS_INLINE void
 ucp_request_complete_tag_recv(ucp_request_t *req, ucs_status_t status)
 {
-    ucs_trace_req("completing receive request %p (%p) "UCP_REQUEST_FLAGS_FMT
+    ucs_trace_req("completing receive request %p (%p) " UCP_REQUEST_FLAGS_FMT
```

Comment:
unrelated? 
(here and below)

Reply:
unfortunately, it is related.
`src/ucp/core/ucp_request.inl` can't be compiled with C++ - to include in gtest